controllers/v1/observationSubmissionsController.js

```diff
@@ -1131,5 +1137,64 @@ module.exports = class ObservationSubmissions extends Abstract {
     })
   }
 
+  /**
+  * @api {get} /assessment/api/v1/observationSubmissions/list/:observationId?entityId:entityId&solutionId:solutionId  List Observation Submissions
+  * @apiVersion 1.0.0
+  * @apiName List Observation Submissions
+  * @apiGroup Observation Submissions
+  * @apiSampleRequest /assessment/api/v1/observationSubmissions/list/5d1a002d2dfd8135bc8e1615?entityId=5cee7d1390013936552f6a8d&solutionId=5d15b0d7463d3a6961f91749
+  * @apiUse successBody
+  * @apiUse errorBody
+  * @apiParamExample {json} Response:
+  * {
+    "message": "Successfully fetched observation submissions",
+    "status": 200,
+    "result": [
+        {
+             "_id": "5d8de379bccbfb51d4450d05",
+            "entityId": "5bfe53ea1d0c350d61b78d0f",
+            "entityExternalId": "1208138",
+            "entityType": "school",
+            "status": "started",
+            "submissionNumber": 1,
+            "updatedAt": "2019-09-27T10:24:57.182Z",
+            "createdAt": "2019-09-27T10:24:57.182Z"
+        }
+    ]
+  }
+
+
+  */
+   /**
+   * List observation submissions
+   * @method
+   * @name list
+   * @param {Object} req - requested data.
+   * @param {String} req.query.entityId - entity id.
+   * @param {String} req.query.solutionId - solution id.
```

Comment:
@aman-tunerlabs - Why we need entityId and solutionId for this API?

Reply:
@akash-above  we need to give a list of submissions for a given entity which belongs to that observation. As for users/programs and users/entities api we are giving only 10 submission per entity if a user wants to see all the submissions for that entities we can use this api.

Reply:
@akash-above we can remove solution Id from the query.